lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -60,9 +60,6 @@ def _can_retry(e: BaseException) -> bool:
         return False
 
 
-pdf_miner_cache = DiskCache(str(Path.home() / ".sycamore/PDFMinerCache"))
```

Comment:
Interesting dilemma here.  Changing to page-at-a-time makes it harder to cache PdfMiner output.  Yet, PdfMiner still represents a significant chunk of work that would benefit from caching.  Is there a way to stream PdfMiner output into the cache and stream it out?  The only approach that comes to mind is disk spooling, which I expect to be faster than PdfMiner itself.

Reply:
Agree that there isn't a clear solution here. I think we should wait on this until after this PR, and decide on an approach for caching for both OCR and PDFMiner.